src/anyio/from_thread.py

```diff
@@ -82,7 +82,6 @@ async def run_async_cm(self) -> Optional[bool]:
             value = await self._async_cm.__aenter__()
         except BaseException as exc:
             self._enter_future.set_exception(exc)
-            raise
```

Comment:
Is this related?

Reply:
Whoa, how did that get into the PR...must've been an artifact of my work on that other PR. Good job noticing this!